processing-service-core/api/swagger/swagger.yaml

```diff
@@ -952,7 +954,123 @@ paths:
           description: The specified Workflow could not be found.
           schema:
             $ref: "#/definitions/errorMessage"    
+  ##########################
+  ##  Default Config Store
+  ##########################
+  /globalConfig/{key}:
+    x-swagger-router-controller: globalConfig
+    parameters:
+      - name: key
+        in: path
+        type: string
+        description: The identifier of global config for tenant.
+        required: true
+    put:
+      tags:
+        - Global Config
+      parameters:
+        - name: default
+          type: string
+          in: query
+          description: The value to use when setting the global config for the tenants key.
```

Comment:
Shorten to 'The value to use when setting the global config'.

This parameter should be in the body as this is a PUT request.